src/modules/investors/views.py

```diff
@@ -159,11 +165,48 @@ def post(self, request: Request, *args, **kwargs) -> Response:
             return Response(serializer.errors, status=400)
 
         amount = serializer.validated_data["amount"]
+        max_amount_per_deposit = 1000
+        max_amount_per_24h = 1000
+
+        if amount <= 0 or amount > max_amount_per_deposit:
+            return Response(
+                {
+                    "status": "error",
+                    "message": (
+                        f"Deposit amount must be between "
+                        f"0.01 and {max_amount_per_deposit}."
+                    ),
+                },
+                status=400,
+            )
 
         user_clerk_id = request.user.id
         investor, _ = Investor.objects.get_or_create(clerk_id=user_clerk_id)
+        deposit_last_24h = (
+            DepositHistory.objects.filter(investor=investor)
+            .deposited_last_24h()
+            .sum_amount()
+        )
+
+        if deposit_last_24h and deposit_last_24h + amount > max_amount_per_24h:
+            return Response(
+                {
+                    "status": "error",
+                    "message": (
+                        f"You can deposit max {max_amount_per_24h} in last 24h. "
+                        f"Available amount: {max_amount_per_24h - deposit_last_24h}"
+                    ),
```

Comment:
since i still need to translate error messages to polish for polish interface version, status in the response is far more meaningful to me than the message, can you differentiate between statuses, like: deposit_amount_error and deposit_frequency_error?